python/lsst/ap/association/diaPipe.py

```diff
@@ -498,14 +507,17 @@ def run(self,
                 "DiaCalculation. This is unexpected behavior and should be "
                 "reported. Exiting.")
 
-        # Force photometer on the Difference and Calibrated exposures using
-        # the new and updated DiaObject locations.
-        diaForcedSources = self.diaForcedSource.run(
-            diaCalResult.diaObjectCat,
-            diaCalResult.updatedDiaObjects.loc[:, "diaObjectId"].to_numpy(),
-            exposure,
-            diffIm,
-            idGenerator=idGenerator)
+        if self.config.doRunForcedMeasurement:
+            # Force photometer on the Difference and Calibrated exposures using
+            # the new and updated DiaObject locations.
+            diaForcedSources = self.diaForcedSource.run(
+                diaCalResult.diaObjectCat,
+                diaCalResult.updatedDiaObjects.loc[:, "diaObjectId"].to_numpy(),
+                exposure,
+                diffIm,
+                idGenerator=idGenerator)
+        else:
+            diaForcedSources = pd.DataFrame(columns=["diaObjectId", "diaForcedSourceId"])
```

Comment:
I don't think this is correct; surely `DiaForcedSourceTask` has columns for the fluxes (at minimum, [these ones](https://github.com/lsst/ap_association/blob/main/python/lsst/ap/association/diaForcedSource.py#L219-L223))? Unfortunately, `DiaForcedSourceTask` doesn't have a mechanism for reporting its output schema, so fixing this may be tricky.

Pinging @parejkoj for a second opinion, since he knows catalog datasets much better than I do.

Reply:
Confirmed that it doesn't break `Apdb.store`, but the dummy catalog gets written to disk. Will fix.

Reply:
Unfortunately, this is a pandas question, and I don't know what our schema requirements are for our pandas outputs, or whether such a thing is even define-able, given pandas' lack of schema. I'd assume it needs at minimum the indexes defined (which I don't think this does?).

Reply:
Reading the dummy catalog from Butler restores the columns but no index.